### PR TITLE
MSVC: avoid "warning C4310: cast truncates constant value"

### PR DESCRIPTION
**Pull request by mikekaganski**

### CODE REVIEW

File: src/lib/StarFileManager.cxx [hunk 1]
```diff
@@ -102,7 +102,7 @@ struct SfxMultiRecord {
       input->seek(pos, librevenge::RVNG_SEEK_SET);
       return false;
     }
-    if (m_zoneType==char(0xff)) {
+    if (m_zoneType=='\xff') {
```
Comment (stbergmann) [inline src/lib/StarFileManager.cxx:105]: a likely better (but also more intrusive, seeing where m_zoneType is passed along) fix would be to change SfxMultiRecord::m_zoneType from 'char' to explicit 'unsigned char'?

Comment (fosnola) [reply src/lib/StarFileManager.cxx:105]: Yes, this seems better.

Note: I will do that this week-end and then do a commit to solve this problem...

Comment (fosnola) [reply src/lib/StarFileManager.cxx:105]: I have replaced 'char' by explicit 'unsigned char' when reading/using the record' types in https://github.com/fosnola/libstaroffice/commit/009037d20522187ce40afeb6fcd8650f7750e26a . Let me know if this is better...

**Comment by fosnola** [comment]
Done

**Pull request closed without merging**